explorer/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Add accessible name to the nav bar button on mobile [#2036]
```

Comment:
Why "on mobile"?

Reply:
We can get rid of it - the idea is that, from the user's perspective, the nav bar toggle is only visible on mobile.

Reply:
ah, ok, leave it then